crates/edr_provider/src/data.rs

```diff
@@ -620,6 +611,16 @@ where
         let is_auto_mining = config.mining.auto_mine;
         let min_gas_price = config.min_gas_price;
 
+        let local_accounts = config
```

Comment:
Previously we were combining the genesis state with the balance of `config.owned_accounts`. We now always require all genesis/fork state modifications to be present in the `genesis_state`.